### PR TITLE
LPS-125688 Add a plus button in Set Default Address modal

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-125688

@drewbrokke @pei-jung

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/edit_account_entry_address.jsp [hunk 1]
```diff
@@ -25,6 +25,15 @@ AccountEntryDisplay accountEntryDisplay = (AccountEntryDisplay)request.getAttrib
...
 
+String[] types = null;
+
+if (Objects.equals("billing", defaultAddressType) || Objects.equals("shipping", defaultAddressType)) {
+	types = new String[] {defaultAddressType, AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING_AND_SHIPPING};
+}
+else {
+	types = new String[] {AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING, AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING_AND_SHIPPING, AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS_TYPE_SHIPPING};
+}
+
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/edit_account_entry_address.jsp:36]: Move this block of code to line 66. We want to declare variable close to its usage.

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/SessionStorageKeys.es.js [hunk 2]
```diff
@@ -12,4 +12,5 @@
  * details.
  */
 
+export const MODAL_STATE_ACCOUNT_ADDRESSES = 'MODAL_STATE_ACCOUNT_ADDRESSES';
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/SessionStorageKeys.es.js:15]: Is this needed? I only see this key being set but not consumed.

**Review by pei-jung — CHANGES_REQUESTED** [review]
@alee8888 do we need to keep the modal window open after a new default address is added? If not, we should remove all the code for the modal state.

**Comment by alee8888** [comment]
Resent here https://github.com/liferay-usm/liferay-portal/pull/194

**Pull request closed without merging**